src/scripts/rspec-test.sh

```diff
@@ -33,12 +33,16 @@ IFS="$old_ifs"
 
 args=()
 
+if [ -n "$PARAM_ORDER" ]; then
+  args+=(--order $PARAM_ORDER)
```

Comment:
```suggestion
  args+=(--order "$PARAM_ORDER")
```

Reply:
I think there might have been a reason I didn't follow the shellcheck suggestion here, I'll have to test that out as this might break things.